### DISCUSSION TITLE
API due_date format. Clarification of what will work

### DISCUSSION THREAD

**Discussion by faspina** [kanboard/kanboard#5493] [Q&A] [answered]
### Checklist

- [X ] I verified that Kanboard is correctly installed
- [X ] I verified that the problem does not come from a plugin
- [ ] I verified that the problem is not already reported
- [X] I understand that Kanboard is in maintenance mode. It doesn't mean it's abandoned, but there is no significant feature development

I have tried every formate for date/time to set the Due_date of a task via API, either via createTask or upDate Task API. 

Application setting for time are M/D/Y  (06/14/2024) 

For time g;i a  (12:00 pm)

I have tried every combo and UNIX time stamp and I cannot get it to set 

### Actual behaviour

API does not change the due_date 

### Expected behaviour

Due Date change for both updateTask and createTask 

### Steps to reproduce

ANy help to clarify how this should work would be appreciated. 

### Configuration

- Kanboard version: 
- Database type and version:
- PHP version:
- OS:
- Browser:

**Comment by fguillot** [comment, marked as answer]
Use this date format: `YYYY-MM-DD HH:MM`, for example: `2024-07-01 14:25`.

```bash
curl \
  -s \
  -u some-user \
  -d '{"jsonrpc": "2.0", "method": "createTask", "id": 1, "params": {"project_id": 1, "title": "Test API", "date_due": "2024-07-01 14:25"}}' \
  http://mykanboard/jsonrpc.php
```

The documentation has been updated:

- https://docs.kanboard.org/v1/api/task_procedures/#createtask
- https://docs.kanboard.org/v1/api/task_procedures/#updatetask
